docs/compatibility.md

```diff
@@ -277,5 +277,73 @@ However, Spark may produce different results for a compiled udf and the non-comp
 
 When translating UDFs to Catalyst expressions, the supported UDF functions are limited:
 
-| Operand type                                                | Operation |
-| ------------------------------------------------------------------- | ------------------|
+| Operand type             | Operation                                                |
+| -------------------------| ---------------------------------------------------------|
+| Arithmetic Unary         | +x                                                       |
+|                          | -x                                                       |
+| Arithmetic Binary        | lhs + rhs                                                |
+|                          | lhs - rhs                                                |
+|                          | lhs * rhs                                                |
+|                          | lhs / rhs                                                |
+|                          | lhs % rhs                                                |
+| Logical                  | lhs && rhs                                               |
+|                          | lhs &#124;&#124; rhs                                     |
+|                          | !x                                                       |
+| Equality and Relational  |lhs == rhs                                                |
+|                          | <                                                        |
```

Comment:
could follow the `lhs OP rhs` example in these binary operations as well.

Reply:
Updated.